src/customfunctions/functions.ts

```diff
@@ -1,5 +1,12 @@
-import * as MyFunctions from "./sample";
+import { add, addAsync, increment } from "./sample";
 
 // To expose the functions to Excel, we need to set the
-//    CustomFunctionsMapping variable with the set of functions
-CustomFunctionMappings = MyFunctions;
+//    CustomFunctionsMapping variable, matching up the
+//    end-user-facing function names from "functions.json"
+//    to the code implementations.
+
+CustomFunctionMappings = {
```

Comment:
I think it is clearer to set each function using CustomFunctionMappings.FUNCTION rather than assign an object, which would clear out any other assignments.